### DISCUSSION TITLE
Possible validation errors due to maintenance of the INSPIRE Registry

### DISCUSSION THREAD

**Discussion by fabiovinci** [INSPIRE-MIF/helpdesk-validator#1137] [Announcements]
Dear community,

due to **maintenance operations of the INSPIRE Registry** ([see related communication](https://github.com/INSPIRE-MIF/helpdesk-registry/discussions/108)) scheduled in the following slots:
- 13 November 2024, 16:00 - 16:30 CET.
- 14 November 2024, 13:00 - 18:00 CET.

the service may be temporarily unavailable, operate more slowly, or have limited functions.

**Please be aware that this may lead to validation errors during this period.**

Apologies for any inconvenience this might cause.

**Comment by jenriquesoriano** [comment]
The maintenance of the INSPIRE Registry has been successfully completed.
If you encounter any issues, please don’t hesitate to contact us.
